tests/test_utils.py

```diff
@@ -0,0 +1,206 @@
+import sys
+
+import pytest
+
+if sys.version_info < (3, 10):
+  pytest.skip('Python 3.10 or higher is required', allow_module_level=True)
```

Comment:
Must be done for our older test matrix as it would fail on parsing the new `UnionType`s